curriculum/dictionaries/spanish/comments.js

```diff
@@ -0,0 +1,454 @@
+  },
+  {
+    id: '5tx4ow',
+    text: 'Sitios de entretenimiento'
+  },
+  {
+    id: '9yu58b',
+    text: 'Ejemplo listas de inventario'
+  },
+  {
+    id: 'ciddtb',
+    text: 'Encuentra el valor del objetivo y su padre'
+  },
+  {
+    id: 'ixx548',
+    text: 'Arregla el código de abajo para que evalúe a true'
+  },
+  {
+    id: '6mbhjj',
+    text: 'Por ejemplo: Redux.createStore()'
+  },
+  {
+    id: 'jshtzq',
+    text:
+      'Función que retorna una cadena de texto representando una taza de té negro'
+  },
+  {
+    id: 'cw1ghf',
+    text:
+      'Función que retorna una cadena de texto representando una taza de té verde'
+  },
+  {
+    id: 'iuccln',
+    text: 'Genera un arreglo lleno al azar'
+  },
+  {
+    id: 'bm2mop',
+    text: 'Obtiene las pestañas después de la pestaña'
+  },
+  {
+    id: 'kchz5k',
+    text: 'Obtiene las pestañas antes de la pestaña'
+  },
+  {
+    id: 'bfd23c',
+    text: `Dada una función (representando el tipo de té) y el número de tazas necesarias, la 
+    siguiente función retorna un arreglo de cadenas de texto (cada una representando un tipo específico de té).`
+  },
+  {
+    id: 'ead98i',
+    text: 'Variable de conteo global:'
+  },
+  {
+    id: '7zf0i2',
+    text: 'Únelos juntos'
+  },
+  {
+    id: '5j2l88',
+    text: 'Vamos a crear tres ventanas del navegador'
+  },
+  {
+    id: 'e843r9',
+    text: 'Abramos una nueva pestaña por ahora'
+  },
+  {
+    id: '5fvehh',
+    text: 'myVar no está definida afuera de myLocalScope'
+  },
+  {
+    id: 'qn720a',
+    text:
+      'Ahora, agrega console.clear() antes de tu console.log() para limpiar la consola del navegador y pasar la prueba'
+  },
+  {
+    id: 'j86mef',
+    text:
+      'Ahora completa la apertura de la pestaña, cierre, y otras operaciones'
+  },
+  {
+    id: 'mk7rvy',
+    text: 'Ahora remueve la línea del console log para pasar la prueba'
+  },
+  {
+    id: 'n7vm1s',
+    text: 'Cambia solo el código encima de esta línea'
+  },
+  {
+    id: 'cvh4x7',
+    text: 'Cambia solo el código debajo de esta línea'
+  },
+  {
+    id: 'lvmnm7',
+    text: 'Abre una nueva pestaña para memes de gatos'
+  },
+  {
+    id: 'avpx79',
+    text: 'Abre la consola de tu navegador.'
+  },
+  {
+    id: '0b5ps6',
+    text: 'Relleno entre el límite del canvas SVG y el gráfico'
+  },
+  {
+    id: 'uemoej',
+    text: 'Inserta n ceros a la fila actual para crear las columnas'
+  },
+  {
+    id: 'lm86nf',
+    text: 'Inserta la fila actual, que ahora contiene n ceros, al arreglo'
+  },
+  {
+    id: 'qscelx',
+    text: 'Los métodos Redux están disponibles desde un objeto Redux'
+  },
+  {
+    id: 'atqiig',
+    text: 'Renderiza un input, button, y ul debajo de esta línea'
+  },
+  {
+    id: 'yq81wg',
+    text: 'Renderiza el Provider debajo de esta línea'
+  },
+  {
+    id: 'kxio9j',
+    text:
+      'responseFromServer es establecido a true para representar una respuesta no satisfactoria del servidor'
```

Comment:
The English text for this key is:
> responseFromServer is set to false to represent an unsuccessful response from a server

I believe you meant to put `false` in this line:
```suggestion
      'responseFromServer es establecido a false para representar una respuesta no satisfactoria del servidor'
```